docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,326 @@
+
+#### Core projects
+
+Core is defined as Zowe projects that are required and are foundational in order to realize the full value of Zowe for everyone including all the other extensions that may depend on them. Configuration and usage is recommended.
+
+Zowe is fully responsible for all Core projects. A non-Zowe project cannot be clasified as a Core project. Each active squad is responsible for the core project with co-responsibility of the TSC.
+
+The [list of current core projects alongside with extension and their lifecycle stage](https://github.com/zowe/community/blob/master/PROJECTS.md) is owned by TSC. 
+
+#### Extension projects
+
+Zowe extension is a project within Zowe that is owned by one of the squads and that isn't part of the Core. As such it doesn't have to follow the version boundaries that the Zowe Core shares. 
+
+The [list of current core projects alongside with extension and their lifecycle stage](https://github.com/zowe/community/blob/master/PROJECTS.md) is owned by TSC. 
+
+#### Plug-in
+
+A more general term used to describe a modular piece of some component. Depending on component or squad context, a plug-in is sometimes referred to as an *app*, *extension*, *plug-in*, etc. A component may have multiple plug-ins, sometimes working together to form a single purpose or user experience, but an individual plug-in belongs to a single component. See [extension](zowe-glossary.md#extension) for additional context.
+
+#### Squad
+  
+A group of people contributing and participating in the Zowe project. Such a group owns one or more projects. Every squad is required to have a representative on the Technical Steering Committee (TSC), and participate in relevant working groups. For more information about how a squad works, see [Squads](https://github.com/Zowe/community/blob/master/Technical-Steering-Committee/squads.md).
+
+#### Zowe Conformance Program
+
+The Zowe Support Provider Conformance Program gives vendors the ability to showcase their Zowe support competencies via well-defined criteria. It is administered by the Linux Foundation and Open Mainframe Project.
+
+#### Zowe Component
+
+Zowe is a collection of both *client* and *server* code. You can install only some of Zowe, or all of it, depending on your needs. Zowe splits the major sections of the code into *components*, with each serving an important purpose.
+  
+Server components are packaged in a standardized way to include all services and plug-ins in one deliverable. Extensions to Zowe can also be delivered as third-party server components. For more information about how these extensions can use a manifest file, see [Zowe component manifest](https://docs.Zowe.org/stable/extend/packaging-zos-extensions/#zowe-component-manifest).
+
+## Current projects
```

Comment:
I feel that here we are heavily duplicating information we have on other places. Can we at least limit the duplication by pointing to the correct places?

It's a case for most concepts from this place forward.

Reply:
Done by my changes.